styles.css

```diff
@@ -979,19 +979,28 @@ ul {
   border: 1px solid #60c095;
 }
 
+input[type="text"],
+input[type="email"] {
+  width: 100%;
+  padding: 16px 15px 16px 15px;
+  margin: 8px 0;
+  box-sizing: border-box;
+}
+
 ::placeholder {
   color: #6b778c;
-  padding: 16px 15px 16px 15px;
-  font-size: 15px;
   line-height: 24px;
   font-weight: 400;
 }
 
-textarea::-webkit-input-placeholder {
+textarea {
+  width: 100%;
+  padding: 16px 15px 16px 15px;
+  margin: 8px 0;
```

Comment:
Kindly adjust the height of the textarea to match the design as closely as possible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/193379533-79964cc2-f2be-47e7-94f1-96047d819172.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/193379522-13310479-9da3-4298-a13c-f007e18e5b44.png)